examples/seismic/tutorials/07_DRP_schemes.ipynb

```diff
@@ -396,18 +396,14 @@
     "        x, z = dimensions\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    ur = Upper(grid=model.grid)
</pre><p>Should these subdomains be created closer to where they are used?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2245/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>I prefer to create them with the <code>Grid</code> and it seems to me to make most sense to make all the grids + subgrids you need before doing anything else, rather than instantiating them as you need them </p>